packages/odyssey-react-mui/README.md

```diff
@@ -47,11 +47,36 @@ import { PasswordInput } from "@okta/odyssey-react-mui";
 Add the Material-UI `ThemeProvider` around your whole app:
 
 ```jsx
-import { ThemeProvider } from "@mui/material/styles";
-import { odysseyTheme } from "@okta/odyssey-react-mui";
+import { ThemeProvider } from "@okta/odyssey-react-mui";
+
+const YourAppRoot = ({ children }) => <ThemeProvider>{children}</ThemeProvider>;
+```
+
+Add Material-UI components not exported from Odyssey with Odyssey styling:
+
+```jsx
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { odysseyTheme, ThemeProvider } from "@okta/odyssey-react-mui";
+
+const YourAppRoot = ({ children }) => (
+  <MuiThemeProvider theme={odysseyTheme}>
+    <ThemeProvider>{children}</ThemeProvider>
+  </MuiThemeProvider>
+);
+```
+
+Add your own theme to MUI (does not affect Odyssey components):
+
+```jsx
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ThemeProvider } from "@okta/odyssey-react-mui";
+
+import { myMaterialUiTheme } from "./myMaterialUiTheme";
 
 const YourAppRoot = ({ children }) => (
-  <ThemeProvider theme={odysseyTheme}>{children}</ThemeProvider>
+  <MuiThemeProvider theme={myMaterialUiTheme}>
+    <ThemeProvider>{children}</ThemeProvider>
```

Comment:
How does this cascade work? I imagine the Odyssey theme will still affect the custom theme passed upon any overlapping theme values?

Reply:
Odyssey's theme should be its own and only affect MUI components in Odyssey. We'll have to test this out, but if you have 2 versions of MUI installed, that's what would happen.

The question is if Yarn would force you to use a single MUI version, then they're shared.

What are your thoughts?

Reply:
> What are your thoughts?

Nothing specific, just was curious if there would be collisions but it sounds like the Odyssey components are namespaced in a way that would make such an event very unlikely.